### ISSUE TITLE
add backdoor path

### ISSUE THREAD

**Issue by enzo123456fes** [NilaierMusic/NavigatingStars#1]
add backdoor path
![QQ截图20240429182925](https://github.com/NilaierMusic/NavigatingStars/assets/54608803/db2743b4-81a4-4a4b-b231-a172bdfccd63)

**Comment by NilaierMusic** [comment]
Yeah, line pointing to the fire exit is also one of the planned features.
